test/Concurrency/custom_executor_enqueue_availability.swift

```diff
@@ -26,10 +26,12 @@ final class OldExecutorOldStdlib: SerialExecutor {
 /// availability, since in this case the UnownedJob version needs to exist.
 @available(SwiftStdlib 5.1, *)
 final class BothExecutorOldStdlib: SerialExecutor {
-  func enqueue(_ job: UnownedJob) {} // expected-note{{'enqueue' declared here}}
+  func enqueue(_ job: UnownedJob) {}
 
+  // This no longer warns, because of the use of StdlibDeploymentTarget in the
```

Comment:
A bit unfortunate

Reply:
Agreed. I couldn't see a good way around that for this test.